wagtail_localize/migrations/0001_initial.py

```diff
@@ -4,52 +4,50 @@
 
 from django.db import migrations, models
 from django.db.migrations.recorder import MigrationRecorder
-from wagtail import VERSION as WAGTAIL_VERSION
 
 
 def get_run_before_and_revision_model():
     # Keep the existing behaviour for Wagtail pre-4.0.
     run_before = []
     revision_model = "wagtailcore.PageRevision"
 
-    if WAGTAIL_VERSION >= (4, 0, 0):
```

Comment:
please leave this as is, otherwise people upgrading pre 4.0 will have loads of issues

Reply:
Reverted main migration